### PR TITLE
Continued adding images and text to the Registration Method Examples

### LINKED ISSUE #1864: Update the Image Registration Method examples to contain images and useful text output

### PR DESCRIPTION
**Pull request by mbopfNIH**
Resolves #1864

### CODE REVIEW

File: Examples/ImageRegistrationMethod3/Documentation.rst [hunk 1]
```diff
@@ -1,11 +1,86 @@
...
 --------
 
+.. include:: registrationExamplePrefix.rst
+
+Example Run
+-----------
+
+Running the Python code with the following inputs:
+
+``main('BrainProtonDensitySliceBorder20.png', 'BrainProtonDensitySliceShifted13x17y.png', 'displaceMeth3.hdf5')``
```
Comment (zivy) [inline Examples/ImageRegistrationMethod3/Documentation.rst:14]: See comment with respect to `Documentation.rst` for method 2.

File: Examples/ImageRegistrationMethod4/Documentation.rst [hunk 2]
```diff
@@ -1,11 +1,86 @@
...
 --------
 
+.. include:: registrationExamplePrefix.rst
+
+Example Run
+-----------
+
+Running the Python code with the following inputs:
+
+``main('BrainProtonDensitySliceBorder20.png', 'BrainProtonDensitySliceShifted13x17y.png', 'displaceMeth4.hdf5')``
```
Comment (zivy) [inline Examples/ImageRegistrationMethod4/Documentation.rst:14]: See comment with respect to `Documentation.rst` for method 2.

**Review by zivy — APPROVED** [review]
Minor comment about the text in the `Documentation.rst` files.

File: Examples/ImageRegistrationMethod4/Documentation.rst (hunk 2 above)
Comment (mbopfNIH) [reply Examples/ImageRegistrationMethod4/Documentation.rst:14]: Agreed. These made more sense at the beginning of my task, but now they are just confusing. Replacing them with a generic version makes more sense. Ideally, there would be a whole usage section, but the code was in too much flux for that to make sense.

**Comment by blowekamp** [comment]
These changes were integrated into PR #1880.

**Pull request closed without merging**